src/widgets/table.rs

```diff
@@ -291,8 +302,8 @@ impl<'a> Table<'a> {
 
     /// If set to `true`, the selection gap will always be allocated, regardless of if something is
     /// selected or not
-    pub fn highlight_set_always_space(mut self, value: bool) -> Self {
-        self.highlight_always_space = value;
+    pub fn highlight_set_selection_space(mut self, value: HighlightSpacing) -> Self {
+        self.highlight_spacing = value;
```

Comment:
Enum is named fine - the method should be more builder-like (so `highlight_spacing()`?) Don't forget to update the doc comment here too.

Reply:
In the doc comment it would be good to note why this feature exists. Sort of reiterate the commit message but reword it so that it fits well with the tone of the docs - i.e. "this works like ..." rather than "this is changed because ..."